mlir-sycl/test/Transforms/inliner.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: sycl-mlir-opt -always-inline -split-input-file -verify-diagnostics %s | FileCheck %s
+
+// CHECK-LABEL: func.func @caller() -> i32 {
+// CHECK-NEXT:    %c1_i32 = arith.constant 1 : i32  
+// CHECK-NEXT:    %0 = sycl.call() {FunctionName = @callee, MangledFunctionName = @callee, TypeName = @A} : () -> i32
+// CHECK-NEXT:    %1 = arith.addi %c1_i32, %0 : i32
+// CHECK-NEXT:    return %1 : i32
+// CHECK-NEXT:  }
+
```

Comment:
Should prob. add a caller that is a gpu.func calling a func.func via a sycl.call operation.